packages-exp/auth-exp/src/api/index.ts

```diff
@@ -88,7 +92,8 @@ export async function performApiRequest<T, V>(
       } else {
         // TODO probably should handle improperly formatted errors as well
         // If you see this, add an entry to SERVER_ERROR_MAP for the corresponding error
-        throw new Error(`Unexpected API error: ${json.error.message}`);
+        console.error(`Unexpected API error: ${json.error.message}`);
```

Comment:
Is this meant to be here?

Reply:
I added the console.error since we never expect to see these, it would help in debugging since we mask json.error.message with an error from the map (in this case, internal-error)